packages/components/multiselect/stories/MultiSelect.stories.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
```

Comment:
This file also needs a renaming to `Multiselect`.

btw, I really thought it should be `MultiSelect` because I consider them two separate words ("multi select" or "multi-select") but there are [examples](https://mdbootstrap.com/docs/standard/extended/multiselect/) on the internet that proofed my wrong 😐

Reply:
yes, same for me, i started out with MultiSelect just to discover that it should be Multiselect, and this file name slipped my renaming.